extensions/extensions-helm/aissemble-trino-chart/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: aissemble-trino-chart
+description: Baseline chart for deploying Trino
+apiVersion: v2
+# The following versions are automatically set by the plugin so these are only a placeholder
```

Comment:
Q: Which plugin is this referring to?

Reply:
I would recommend adding helm-maven-plugin in place of "plugin" to make it more specific.  I had a moment of "is this a new trino plugin"